packages/hotspot/configure/src/hotspot-polygon.jsx

```diff
@@ -124,10 +132,11 @@ class PolComponent extends React.Component {
       return acc;
     }, []);
 
```

Comment:
I guess this could have been simplified like:

this.setState({ points: newPoints, ...this.getOffset(newPoints), isDragging: !!updateModel });

*This doesn't work if you want to update isDragging ONLY IF updateModel is true, so check if the logic makes sense.